manifests/m/Metanorma/Metanorma/1.11.1/Metanorma.Metanorma.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.6.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Metanorma.Metanorma
+PackageVersion: 1.11.1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: metanorma.exe
```

Comment:
```suggestion
- RelativeFilePath: metanorma-windows-x86_64.exe
```

Reply:
@stephengillie thanks for spotting this, and sorry for the inconvenience there was an issue with the package itself.

Just reuploaded the package

Is it possible to rerun the ci?

Reply:
upd, looks like it's in progress already after update of the sha256